src/htcondorce/rrd.py

```diff
@@ -6,7 +6,7 @@
 # Try using rrdtool Python bindings
 try:
     import rrdtool
-except ImportError:
+except ModuleNotFoundError:
```

Comment:
nothing wrong with this, although ImportError continues to work in Python 3

Reply:
Yup, see https://github.com/htcondor/htcondor-ce/pull/406#discussion_r564889353